subsys/fs/Kconfig.fatfs

```diff
@@ -129,6 +129,13 @@ config FS_FATFS_MAX_SS
 	range 512 4096
 	default 512
 
+config FS_FATFS_WIN_ALIGNED
+	int "Memory alignment for the member \"win\" in FATFS"
+	default 0
```

Comment:
I am not sure whether 0 is correct alignment, shouldn't be 1 as default? There is no point to align anything to 0.

Reply:
Hi Ermel,

Yup, it shouldn't align with 0 although I use the 0 as a switch to detect for :no alignment", which is not a good implementation. Will default the value to 1 and resubmit the PR.